CHANGELOG.md

```diff
@@ -42,3 +42,42 @@ to work with various other command line dispatchers such as LSF. See the
 
 * On the JES and SFS backends, task `command` blocks are now always
 passed absolute paths for input `File`s.
+
+* On the SFS backends, the call directory now contains two sub-directories:
+    * `inputs` contains all the input files that have been localized for this task (see next below for more details)
+    * `execution` contains all other files (script, logs, rc, potential outputs etc...)
+    
+* On the SFS backends, inputs are now localized as follow:
+    The absolute path of the directory containing the input is hashed.
+    This hash is used to create a directory under `inputs` where the file will be localized
+    e.g:
+    
+    ```
+    ├── execution
+    │   ├── rc
+    │   ├── script
+    │   ├── script.background
+    │   ├── script.submit
+    │   ├── stderr
+    │   ├── stderr.background
+    │   ├── stdout
+    │   └── stdout.background
+    └── inputs
+        ├── 2c142eed1d519eabc0520becff443327
```

Comment:
seems reasonable to me.  The only thing would be to not use hashes in the inputs subdirectory, and use some other form of name-shortening (like take the last element of the path plus an incrementing number after 1 like when you download the same file in chrome twice).  

Generally though, I think people don't spelunk as much on inputs as they do on outputs, so this seems fine